packages/@lightningjs/ui-components/src/components/Control/Control.d.ts

```diff
@@ -17,26 +17,63 @@
  */
 
 import type lng from '@lightningjs/core';
-import { ButtonSmall, ButtonStyle } from '../Button';
-import type { StylePartial } from '../../types/lui';
+import ButtonSmall, { ButtonSmallStyle } from '../Button/ButtonSmall';
+import Button from '../Button';
+import { StylePartial } from '../../types/lui';
 
 type LogoStyleObject = {
   radius: lng.Tools.CornerRadius;
   h: number;
   w: number;
 };
 
-export type ControlStyle = ButtonStyle & {
-  h: number;
+type ControlStyle = ButtonSmallStyle & {
   iconStyle: Record<string, unknown>;
   logoStyle: LogoStyleObject;
   radius: lng.Tools.CornerRadius;
 };
 
-export default class Control extends ButtonSmall {
+declare namespace Control {
+  // TODO: need to replace Button with ButtonSmall here , but throws error. Is this correct now?
```

Comment:
@anthony9187 Need your eyes on this comment

Reply:
is this an old comment from when we were extending button instead?